README.md

```diff
@@ -16,16 +16,27 @@ The goal of this library is to give the user the ability to efficiently train De
 
 ## Local setup
 
-### Prerequisites
+### Using [Anaconda](http://anaconda.org) (recommended)
+
+- [Install Anaconda](https://www.anaconda.com/download/) or [Miniconda](https://conda.io/miniconda.html) for your platform
+- Install our OpenMined Unity app by following the guidelines [here](https://github.com/OpenMined/OpenMined)
+- Install PySyft
+  - Linux: run `./install_for_anaconda.sh`
+  - Windows: execute `https://conda.io/miniconda.html`
+
```

Comment:
Link to tutorials instead, I think. https://github.com/OpenMined/tutorails

Reply:
your link is broken! https://github.com/OpenMined/tutorials

Reply:
Whoops!